src/includes.h

```diff
@@ -11,7 +11,15 @@
 #include <linux/module.h>
 #include <linux/moduleparam.h>
 #include <linux/blkdev.h>
-#include <linux/genhd.h>
+// 'genhd.h' has been removed and 'gendisk' struct has been moved to the 'blkdev.h' in the kernel 5.18.
+// Old compilers may not have '__has_include' macros, but 'genhd.h' exists on those systems.
+#if defined __has_include
+#	if __has_include (<linux/genhd.h>)
```

Comment:
Please delete tabs around this place

Reply:
It seems the indents are used in some places in the Linux kernel source code like [here](https://github.com/torvalds/linux/blob/ed4643521e6af8ab8ed1e467630a85884d2696cf/tools/include/asm/barrier.h#L36-L46)
```C
#ifndef smp_rmb
# define smp_rmb()	rmb()
#endif

#ifndef smp_wmb
# define smp_wmb()	wmb()
#endif

#ifndef smp_mb
# define smp_mb()	mb()
#endif
```
or [here](https://github.com/torvalds/linux/blob/2a987e65025e2b79c6d453b78cb5985ac6e5eb26/tools/build/feature/test-all.c#L13-L19)
```C
#define main main_test_libpython
# include "test-libpython.c"
#undef main

#define main main_test_libperl
# include "test-libperl.c"
#undef main
```
or [here](https://github.com/torvalds/linux/blob/e2b542100719a93f8cdf6d90185410d38a57a4c1/arch/arm/include/asm/glue-cache.h#L33-L47)
```C
#if defined(CONFIG_CPU_FA526)
# ifdef _CACHE
#  define MULTI_CACHE 1
# else
#  define _CACHE fa
# endif
#endif


#if defined(CONFIG_CPU_ARM926T)
# ifdef _CACHE
#  define MULTI_CACHE 1
# else
#  define _CACHE arm926
# endif
#endif
```
So I'm changing tabs to spaces still for better readability.
Also there is an example of the `__has_include` operator from GCC docs with the similar indents https://gcc.gnu.org/onlinedocs/cpp/_005f_005fhas_005finclude.html